cmd/srcd/cmd/components.go

```diff
@@ -145,10 +132,71 @@ var componentsInstallCmd = &cobra.Command{
 	},
 }
 
+// componentsStartCmd represents the components start command
+var componentsStartCmd = &cobra.Command{
+	Use:   "start [component]",
```

Comment:
Given that we can actually pass multiple components I'd change this:
```suggestion
	Use:   "start component [component...]",
```

Reply:
`Use` of the start command matches the `Use` of install command.

Reply:
done